docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: docker/Dockerfile
+    ports:
+      - "8000:8000"
```

Comment:
@vasylboliuk could you please add an endline

Reply:
Done